examples/mnist/mnist_deep_simplified_distributed.py

```diff
@@ -0,0 +1,272 @@
+# Copyright 2015 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""A simplified deep MNIST classifier using convolutional layers.
```

Comment:
Since this file is modified by us - need to add the following:

1. Mention the source of this file. 
2. Intel copyright header

Reply:
Mentioned the source of the file and added Intel copyright header. Could you please take another review on this?